### PR TITLE
Docs: Define npm install by quinoa

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/modules/ROOT/pages/advanced-guides.adoc [hunk 1]
```diff
@@ -39,10 +39,12 @@ Quinoa can be configured to install NodeJS and NPM in the project directory:
 ----
 quarkus.quinoa.package-manager-install=true <1>
 quarkus.quinoa.package-manager-install.node-version=18.18.0 <2>
+quarkus.quinoa.package-manager-install.npm-version=10.1.0 <3>
 ----
 
 <1> Enable package manager install
 <2> Define the version of NodeJS to install
+<3> Define the version of Npm to install
```
Comment (melloware) [inline docs/modules/ROOT/pages/advanced-guides.adoc:47]:
```suggestion
<3> Define the version of NPM to install
```

**Review by melloware — CHANGES_REQUESTED** [review]

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
@shivam-sharma7 can you translate "Now" in Hindi for me?  For example if `"today": "आज",` is for Today what about the Current Time.  In english we say the time "NOW" is.  I am updating my PF translations

**Comment by shivam-sharma7** [comment]
@melloware `अभि`

**Comment by melloware** [comment]
Thank you!

**Pull request merged**